apistar/exceptions.py

```diff
@@ -26,7 +26,7 @@ class Found(APIException):
     default_status_code = 302
     default_detail = ''
 
-    def __init__(self, location, *args, **kwargs):
+    def __init__(self, location: str, *args: Any, **kwargs: Any) -> None:
```

Comment:
I'd suggest that we get rid of `*args, **kwargs`, and explicitly use `detail` and `status_code` instead.